### DISCUSSION TITLE
Welcome to Ambianic UI Discussions!

### DISCUSSION THREAD

**Discussion by ivelin** [ambianic/ambianic-ui#462] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.
  * When you identify a bug or a feature request, [open an issue](https://github.com/ambianic/ambianic-ui/issues).

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by jjsenay** [comment]
How does the Ambianic Edge stay updated?

**Comment by jjsenay** [comment]
A new release on github, pushes a new docker image. Edge uses watchtower to watch for and pull new docker images when available.

**Reply by ivelin** [reply]
This is all correct. Although I just realized that you posted comments to the welcome topic. Maybe re-post as a separate Q&A at the top Discussions level: https://github.com/ambianic/ambianic-ui/discussions
